pqrs.md

```diff
@@ -59,9 +61,22 @@ LOWEST -> LOW -> NORMAL -> HIGH -> HIGHEST -> MONITOR
 Therefore, it is RECOMMENDED that plugins that plugins modifying events in a specific manner (e.g. handling specific block interaction in a sign-clicking plugin) use LOW priority, while plugins modifying events in a large scale (e.g. area protection plugins cancelling all block interaction in an area) use HIGH priority.
 
 ## 6. Custom events
-Custom events SHOULD extend `pocketmine\event\plugin\PluginEvent`, preferrably with a superclass for all events from the same plugin.
+* Custom events SHOULD extend `pocketmine\event\plugin\PluginEvent`, preferrably with a superclass for all events from the same plugin.
 
-## ∞. Contact / Comments
-For any discussion, visit [the forum thread](https://forums.pmmp.io/threads/pqrs.855/).
+## 7. Library usage
+* When adding libraries to plugins, the library classes SHOULD be shaded.
```

Comment:
Note that when Poggit supports composer library inclusion, it may shade the libraries automatically according to the PSR-4 declaration and/or AST analysis.